.github/workflows/lint.yml

```diff
@@ -6,4 +6,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: gnosis/safe-react-eslint-plus-action@v4.0.0
+      - uses: gnosis/safe-react-eslint-plus-action@node-update
```

Comment:
We should remember to change this to the correct version once the action is merged